packages/vinxi/lib/plugins/routes.js

```diff
@@ -1,4 +1,4 @@
-import { relative } from "node:path";
+import { relative } from "pathe";
```

Comment:
Manifest lookups would fail because of `\`.
https://github.com/nksaraf/vinxi/blob/505dd51fd130dc6747de2ff5259ef0cf86f62b6f/packages/vinxi/lib/plugins/routes.js#L77